runtime-transaction/src/runtime_transaction.rs

```diff
@@ -64,7 +64,13 @@ impl<T: StaticMetaAccess> StaticMeta for RuntimeTransaction<T> {
     }
 }
 
-impl<M: DynamicMetaAccess> DynamicMeta for RuntimeTransaction<M> {}
+impl<T: DynamicMetaAccess> DynamicMeta for RuntimeTransaction<T> {}
```

Comment:
Only changed `M` to `T` here to be consistent with the other generic parameter naming in this file.